platforms/nuttx/src/px4/stm/stm32_common/version/board_mcu_version.c

```diff
@@ -151,8 +153,10 @@ int board_mcu_version(char *rev, const char **revstr, const char **errata)
 		chip_errata = NULL;
 		break;
 
-	case MCU_REV_STM32F7_REV_V:
-		*rev = 'V';
+	case MCU_REV_STM32F4_REV_5:
```

Comment:
Are these gaps we could be preemptively filling in before someone hits them?

Reply:
I'm sorry, I could not quite understand what you've meant there. Could you elaborate?

Reply:
@dagar I believe it is a sparse array. 

This is the latese published in R18 of https://www.st.com/resource/en/reference_manual/dm00031020-stm32f405-415-stm32f407-417-stm32f427-437-and-stm32f429-439-advanced-arm-based-32-bit-mcus-stmicroelectronics.pdf
```
0x1000 = Revision A
0x1003 = Revision Y
0x1007 = Revision 1
0x2001= Revision 3
0x2003= Revision 5 and B
```